Crash Report 3.0.3: MissingResourceException

**Describe the bug**
```
Caused by: java.util.MissingResourceException: 

 
  at java.util.Locale.getISO3Language (Locale.java:1702)

 
  at org.kiwix.kiwixmobile.utils.LanguageUtils.<clinit> (LanguageUtils.java:156)

 
  at org.kiwix.kiwixmobile.utils.LanguageUtils.handleLocaleChange (LanguageUtils.java)

 
  at org.kiwix.kiwixmobile.base.BaseActivity.onCreate (BaseActivity.java:44)

 
  at org.kiwix.kiwixmobile.splash.SplashActivity.onCreate (SplashActivity.java:33)
```


**Expected behavior**
No crash


**Environment**
- Version of Kiwix Android : 3.0.3 (6230003)
- Device : HTC HTC U11+ (htc_ocmdugl), 
- OS version : Android 9

--- a/app/src/main/java/org/kiwix/kiwixmobile/utils/LanguageUtils.kt
+++ b/app/src/main/java/org/kiwix/kiwixmobile/utils/LanguageUtils.kt
@@ -36,6 +36,7 @@ import org.kiwix.kiwixmobile.utils.Constants.TAG_KIWIX
 import org.kiwix.kiwixmobile.utils.files.FileUtils
 import java.text.Collator
 import java.util.Locale
+import java.util.MissingResourceException
 
 class LanguageUtils(private val context: Context) {
   private val localeLanguageCodes: List<String> = languageCodesFromAssets()
@@ -152,8 +153,14 @@ class LanguageUtils(private val context: Context) {
 
   companion object {
 
-    private var localeMap =
-      Locale.getAvailableLocales().associateBy { it.isO3Language.toUpperCase(Locale.ROOT) }
+    private var isO3LanguageToLocaleMap: Map<String, Locale> =
+      Locale.getAvailableLocales().associateBy {
+        try {
+          it.isO3Language.toUpperCase(Locale.ROOT)
+        } catch (ignore: MissingResourceException) {
+          it.language.toUpperCase(Locale.ROOT)
+        }
+      }
 
     private var fontExceptions = mapOf(
       "km" to "fonts/KhmerOS.ttf",
@@ -204,7 +211,7 @@ class LanguageUtils(private val context: Context) {
      */
     @JvmStatic
     fun iSO3ToLocale(iso3: String?): Locale? =
-      iso3?.let { localeMap[it.toUpperCase(Locale.ROOT)] }
+      iso3?.let { isO3LanguageToLocaleMap[it.toUpperCase(Locale.ROOT)] }
 
     @JvmStatic
     fun getCurrentLocale(context: Context) = context.locale

